renderer/plugin.js

```diff
@@ -138,6 +138,47 @@ org.ekstep.contentrenderer.questionUnitPlugin = Plugin.extend({
    */
   getQuestionState: function() {
     return this._question.state;
+  },
+  /**
+   * provide media url to asset, runs inside the context of [mtf, fib, mcq context]
+   * @memberof org.ekstep.questionunit
+   * @param {String} url from question set.
+   * @returns {String} url.
+   */
+  getAssetUrl:function(url){
+    if(isbrowserpreview){// eslint-disable-line no-undef
+      return url;
+    }
+    else{
+      return 'file:///' + EkstepRendererAPI.getBaseURL()+ url;
+    }
+  },
+  /**
+   * play audio once at a time, runs inside the context of [mtf, fib, mcq context]
+   * @memberof org.ekstep.questionunit
+   * @param {String} audio from question set.
+   */
+  playAudio: function(audio){
```

Comment:
Expose Play and Pause as a different methods. So that plugin can handle to play & pause that audios. The base plugin(questionUnit) can't handle all the use cases.

Take 'audio' as an input function parameter.

Reply:
Currently I am discussing it with @ramj-funtoot, Will update on this once done